docs/reference/sciencedirect/ArticleMetadata.rst

```diff
@@ -0,0 +1,49 @@
+pybliometrics.sciencedirect.ArticleMetadata
+===========================================
+
+`ArticleMetadata()` implements the `ScienceDirect Article Metadata API <https://dev.elsevier.com/documentation/ArticleMetadataAPI.wadl>`_.
+
+The Article Metadata API allows users to search for documents using a Boolean syntax. It offers two views: 'STANDARD' and 'COMPLETE', with the 'COMPLETE' view providing more detailed data.
+
+For a guide on how to query, check the `ScienceDirect Article Metadata Guide <https://dev.elsevier.com/sd_article_meta_tips.html>`_.
+
+.. currentmodule:: pybliometrics.sciencedirect 
+.. contents:: Table of Contents
+    :local:
+
+Documentation
+-------------
+
+.. autoclass:: ArticleMetadata
+   :members:
+   :inherited-members:
+
+Examples
```

Comment:
Please add that pandas Dataframe can be used. Check the Scopus Search for an example.